src/ProductConstructionService/ProductConstructionService.Api/Dockerfile

```diff
@@ -30,9 +30,11 @@ RUN dotnet restore "./ProductConstructionService/ProductConstructionService.Api/
 
 COPY ["src/Maestro/Client/src", "./Maestro/Client/src"]
```

Comment:
Why don't we just `COPY` the entire `src/Maestro/` directory? I'm guessing its because we don't use all of the projects under that path, but is it worth having multiple `COPY` directives / layers?

Reply:
We do not want to copy the whole SF cluster app. We do not want to copy Maestro.Web (the current BarViz), these are huge.

It's always better to be explicit than to create a monster of a package where you have no idea if things are relevant. It would be then very hard to deprecate things.